demo/clojure/gossip.clj

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bb
+
+(ns maelstrom.gossip
+  (:gen-class)
+  (:require
+    [cheshire.core :as json]
+    [clojure.java.io :as io]))
+
+
+;;;;;;;;;;;;;;;;;;; Util functions ;;;;;;;;;;;;;;;;;;;
+
+;;;;;; Input pre-processing functions ;;;;;;
+
+(defn- read-file
+  "Read a file into a vector of strings.
+  This is used for local testing in repl"
+  [f]
+  (with-open [rdr (io/reader (io/input-stream f))]
+    (reduce conj [] (line-seq rdr))))
+
+
+(defn- process-stdin
+  "Read lines from the stdin and calls the handler"
+  [handler]
+  (doseq [line (line-seq (java.io.BufferedReader. *in*))]
+    (handler line)))
+
+
+(defn- parse-json
+  "Parse the received input as json"
+  [input]
+  (try
+    (json/parse-string input true)
+    (catch Exception e
+      nil)))
+
+
+;;;;;; Output Generating functions ;;;;;;
+
+(defn- generate-json
+  "Generate json string from input"
+  [input]
+  (when input
+    (json/generate-string input)))
+
+
+(let [l ""]
```

Comment:
Is this a Babashka-ism? You might see false locking if this "" and the later "" are interned to the same object, no?

Reply:
I can change this to `(Object.)`